prow/gerrit/adapter/adapter.go

```diff
@@ -332,9 +374,13 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 
 	switch change.Status {
 	case client.Merged:
-		// TODO: Do we want to add support for dynamic postsubmits?
-		postsubmits := c.config().PostsubmitsStatic[cloneURI.String()]
-		postsubmits = append(postsubmits, c.config().PostsubmitsStatic[cloneURI.Host+"/"+cloneURI.Path]...)
+		postsubmits, err := cache.GetPostsubmits(trimmedHostPath, func() (string, error) { return baseSHA, nil })
+		if err != nil {
+			//TODO(mpherman): Return Error once we know this usually works
+			logger.Warn("Failed to get cached InRepoConfig for Postsubmits")
+			postsubmits = append(postsubmits, c.config().PostsubmitsStatic[trimmedHostPath]...)
```

Comment:
I think we need to always append the jobs from `PostsubmitsStatic[trimmedHostPath]`, not only when in repo config fails. We must not have a unit test to confirm that jobs under the `trimmedHostPath` identifier are included, that would be nice to add.
This looks like it is here to find jobs that omit the `https://` prefix.

This also applies to presubmit lookup.